.storybook/preview.tsx

```diff
@@ -2,16 +2,24 @@ import {PortalProvider} from '@gorhom/portal';
 import React from 'react';
 import Onyx from 'react-native-onyx';
 import {SafeAreaProvider} from 'react-native-safe-area-context';
-import ComposeProviders from '../src/components/ComposeProviders';
-import HTMLEngineProvider from '../src/components/HTMLEngineProvider';
-import {LocaleContextProvider} from '../src/components/LocaleContextProvider';
-import OnyxProvider from '../src/components/OnyxProvider';
-import {EnvironmentProvider} from '../src/components/withEnvironment';
-import {KeyboardStateProvider} from '../src/components/withKeyboardState';
-import {WindowDimensionsProvider} from '../src/components/withWindowDimensions';
-import ONYXKEYS from '../src/ONYXKEYS';
+import ComposeProviders from '@src/components/ComposeProviders';
+import HTMLEngineProvider from '@src/components/HTMLEngineProvider';
+import {LocaleContextProvider} from '@src/components/LocaleContextProvider';
+import OnyxProvider from '@src/components/OnyxProvider';
+import {EnvironmentProvider} from '@src/components/withEnvironment';
+import {KeyboardStateProvider} from '@src/components/withKeyboardState';
+import {WindowDimensionsProvider} from '@src/components/withWindowDimensions';
+import ONYXKEYS from '@src/ONYXKEYS';
 import './fonts.css';
 
+type Parameter = {
```

Comment:
I think there is a built in Storybook type for that 
`import { Parameters } from '@storybook/addons';`